forc-plugins/forc-client/src/op/call/transfer.rs

```diff
@@ -0,0 +1,114 @@
+use anyhow::anyhow;
+use fuel_abi_types::abi::program::ProgramABI;
+use fuels::accounts::{wallet::WalletUnlocked, Account};
+use fuels_core::types::{transaction::TxPolicies, Address, AssetId};
+use sway_core;
+
+pub async fn transfer(
+    wallet: &WalletUnlocked,
+    recipient: Address,
+    amount: u64,
+    asset_id: AssetId,
+    tx_policies: TxPolicies,
+    show_receipts: bool,
+    node: &crate::NodeTarget,
+) -> anyhow::Result<super::CallResponse> {
+    println!(
+        "\nTransferring {} 0x{} to address 0x{}...\n",
+        amount, asset_id, recipient
+    );
+
```

Comment:
should we check that there is sufficient balance in the wallet to cover the transfer before calling

Reply:
For a check we would need to perform an additional RPC request to get the balance.

The SDK function should throw an error:
```
error: Failed to transfer funds: provider: io error: Response errors; the target cannot be met due to no coins available or exceeding the 255 coin limit.
```

I think this is somewhat sufficient (`due to no coins available` part); if not - can perform an additional request to validate.
Wdyt?

Reply:
fair enough, this error response sounds reasonable. No need to introduce another RPC request is this case.